.github/workflows/release.yml

```
@@ -63,3 +63,13 @@ jobs:
           name: ${{ steps.changelog.outputs.tag }}
           body: ${{ steps.changelog.outputs.clean_changelog }}
           prerelease: true
+
+      - name: Notify Discord about Pre-Release
+        if: ${{ steps.changelog.outputs.skipped == 'false' }}
+        uses: appleboy/discord-action@master
```

Lets use version instead of referring to branch

```suggestion
        uses: appleboy/discord-action@v1.2.0
```